docs/reference/run-elasticsearch-locally.asciidoc

```diff
@@ -9,163 +15,75 @@
-While this approach is convenient for experimenting and learning, you should never run Elasticsearch in this way in a production environment.
+Refer to <<elasticsearch-intro-deploy, deployment options>> for a list of production deployment options.
 ====
 
-Follow this tutorial if you want to quickly set up {es} in Docker for local development or testing.
+Quickly set up {es} and {kib} in Docker for local development or testing, using the https://github.com/elastic/start-local?tab=readme-ov-file#-try-elasticsearch-and-kibana-locally[`start-local` script].
 
-This tutorial also includes instructions for installing {kib}. 
- If you don't need access to the {kib} UI, then you can skip those instructions.
+This setup comes with a one-month trial of the Elastic *Platinum* license.
+After the trial period, the license reverts to *Free and open - Basic*.
+Refer to https://www.elastic.co/subscriptions[Elastic subscriptions] for more information.
 
 [discrete]
 [[local-dev-prerequisites]]
 === Prerequisites
 
 If you don't have Docker installed, https://www.docker.com/products/docker-desktop[download and install Docker Desktop] for your operating system.
 
-[discrete]
-[[local-dev-env-vars]]
-=== Set environment variables
-
-Configure the following environment variables.
-
-[source,sh]
-----
-export ELASTIC_PASSWORD="<ES_PASSWORD>"  # password for "elastic" username
-export KIBANA_PASSWORD="<KIB_PASSWORD>"   # Used _internally_ by Kibana, must be at least 6 characters long
-----
+- Works on Linux and macOS
+- Works on Microsoft Windows using https://learn.microsoft.com/en-us/windows/wsl/install[Windows Subsystem for Linux (WSL)]
 
 [discrete]
-[[local-dev-create-docker-network]]
-=== Create a Docker network
+[[local-dev-quick-start]]
+=== Run `start-local`
 
-To run both {es} and {kib}, you'll need to create a Docker network:
+To set up {es} and {kib} locally, run the `start-local` script:
 
 [source,sh]
 ----
-docker network create elastic-net
+curl -fsSL https://elastic.co/start-local | sh
 ----
+// NOTCONSOLE
 
-[discrete]
-[[local-dev-run-es]]
-=== Run {es}
-
-Start the {es} container with the following command:
+This script creates an `elastic-start-local` folder containing configuration files and starts both {es} and {kib} using Docker.
 
-ifeval::["{release-state}"=="unreleased"]
-WARNING: Version {version} has not yet been released.
-No Docker image is currently available for {es} {version}.
-endif::[]
+After running the script:
 
-[source,sh,subs="attributes"]
-----
-docker run -p 127.0.0.1:9200:9200 -d --name elasticsearch --network elastic-net \
-  -e ELASTIC_PASSWORD=$ELASTIC_PASSWORD \
-  -e "discovery.type=single-node" \
-  -e "xpack.security.http.ssl.enabled=false" \
-  -e "xpack.license.self_generated.type=trial" \
-  {docker-image}
-----
-
-[discrete]
-[[local-dev-run-kib]]
-=== Run {kib} (optional)
+* *{es}* will be running at http://localhost:9200
+* *{kib}* will be running at http://localhost:5601
```

Comment:
Suggest avoiding this construction (intro sentence should be a complete sentence).

Would prefer the endpoint info to be a little more concise. will leave this alone for now

```
* *{es}*: http://localhost:9200
* *{kib}*: http://localhost:5601
```